packages/client-twitter/src/index.ts

```diff
@@ -13,7 +13,7 @@ class TwitterManager {
     constructor(runtime: IAgentRuntime) {
         this.client = new ClientBase(runtime);
         this.post = new TwitterPostClient(this.client, runtime);
-        // this.search = new TwitterSearchClient(runtime); // don't start the search client by default
+        this.search = new TwitterSearchClient(this.client, runtime);
```

Comment:
please don't enable this by default, as the note further down says it has high risk